code/datums/perks/racial.dm

```diff
@@ -72,9 +72,9 @@
 	user.reagents.add_reagent("adrenol", 5)
 	return ..()
 
-/datum/perk/gutsandglory
-	name = "Guts and Glory"
-	desc = "You pull what made your ancestors conquer the stars from your will, letting your body recover somewhat from any internal damage at the cost of becoming exhausted while it works."
+/datum/perk/iwillsurvive
+	name = "Will to Survive"
```

Comment:
DO NOT
```suggestion
	name = "Guts and Glory"
```
DO NOT

Reply:
I will

> Lets not corrupt player data

How does this _"corrupt player data"_? This shouldn't